tests/ui/invalid_null_ptr_usage.rs

```diff
@@ -25,11 +25,6 @@ fn main() {
 
         let _a: A = std::ptr::replace(std::ptr::null_mut(), A);
 
-        let _slice: *const [usize] = std::ptr::slice_from_raw_parts(std::ptr::null(), 0);
```

Comment:
This shouldn't be removed, this should end up checking that it doesn't lint instead